Sources/SwiftDocC/Infrastructure/Link Resolution/PathHierarchy+Dump.swift

```diff
@@ -21,20 +23,25 @@ private extension PathHierarchy.Node {
     /// Maps the path hierarchy subtree into a representation that can be printed into a visual form for debugging.
     func dumpableNode() -> DumpableNode {
         // Each node is printed as 3-layer hierarchy with the child names, their kind disambiguation, and their hash disambiguation.
+        
+        // One layer for the node itself that displays information about the symbol
         return DumpableNode(
-            name: symbol.map { "{ \($0.identifier.precise) : \($0.kind.identifier.identifier) [\(languages.map(\.name).joined(separator: ", "))] }" } ?? "[ \(name) ]",
-            children: children.sorted(by: \.key).map { (key, disambiguationTree) -> DumpableNode in
+            name: symbol.map(describe(_:)) ?? "[ \(name) ]",
```

Comment:
Yes, that's basically the same, or at least the compiler should be able to make them the same. 

Here `describe(_:)` is referring to [private func describe(_ symbol: SymbolGraph.Symbol) -> String][1] (on line 54 below) by its name. The underscore in the name just because the parameter doesn't have an external argument label.



[1]: https://github.com/swiftlang/swift-docc/pull/643/files#diff-efd1b7cf023713dbd75be874c85bbc4cce9f1f563b748548a39cf68a0bd037c2R54